pkg/reconciler/taskrun/resources/taskresourceresolution.go

```diff
@@ -17,11 +17,7 @@ limitations under the License.
 package resources
 
 import (
-	"errors"
-
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
-	resourcev1alpha1 "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // ResolvedTaskResources contains all the data that is needed to execute
```

Comment:
NIT: ResolvedTaskResources could be renamed to ResolvedTask but we don't have to

Reply:
renamed and updated in the commit message:)